sympy/core/tests/test_basic.py

```diff
@@ -36,6 +36,18 @@ def test_equality():
     assert not(Basic() == 0)
 
 
+def test_compare_issue_9464_a():
+    assert(Basic(1).compare(Basic(None)) > 0)
+
+
+def test_compare_issue_9464_b():
+    assert(Basic(None).compare(Basic(1)) < 0)
+
+
+def test_compare_issue_9464_c():
+    assert(Basic(None).compare(Basic(None)) == 0)
+
+
```

Comment:
No need for 3 separate tests here, you can group them all under `test_compare_issue_9464`